src/common/config_parser.cpp

```diff
@@ -413,8 +414,12 @@ void ConfigParser::addOptionsTraining(cli::CLIWrapper& cli) {
   cli.add<bool>("--multi-node-overlap",
      "Overlap model computations with MPI communication",
      true);
+
   // add ULR settings
   addSuboptionsULR(cli);
+
+  cli.add<std::vector<std::string>>("--task",
```

Comment:
Another proposition for the name was "--defaults-for-task". I am conflicted.

Reply:
My vote for `--task` as it's shorter.